File: alyra1_1_1/alyra1_1_4.py
import math
class Cercle:
	def __init__(self, rayon):
		self.rayon = rayon

	def aire(self):
		return self.rayon**2*math.pi

	def perimetre(self):
		return 2*self.rayon*math.pi

c = Cercle(5)
print({'aire': c.aire(), 'perimetre': c.perimetre()})

File: alyra1_1_1/alyra1_1_4.js
class Cercle {
	constructor(rayon) {
		this.rayon = rayon;
	}
	aire() {
		return Math.pow(this.rayon,2)*Math.PI;
	}
	perimetre() {
		return 2*Math.PI*this.rayon
	}
}


let c = new Cercle(5);
console.log({'aire': c.aire(), 'perimetre': c.perimetre()});
